internal/agentdeployer/_static/docker-agent-base.yml.tmpl

```diff
@@ -24,5 +24,11 @@ services:
       - KIBANA_HOST=https://kibana:5601
       - FLEET_TOKEN_POLICY_NAME=${FLEET_TOKEN_POLICY_NAME}
     volumes:
-      - ${SERVICE_LOGS_DIR}:/tmp/service_logs/
       - ${LOCAL_CA_CERT}:/etc/ssl/certs/elastic-package.pem
```

Comment:
Do you know what is the default type when using this syntax?

Reply:
Just checked in a local run, and the type is also "bind", but the Mode changes from "rw" to empty string. I haven't found what that means exactly... 

```json
        "Mounts": [
            {
                "Type": "bind",
                "Source": "/home/mariorodriguez/.elastic-package/profiles/default/certs/ca-cert.pem",
                "Destination": "/etc/ssl/certs/elastic-package.pem",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/home/mariorodriguez/.elastic-package/tmp/service_logs/agent-sophos-utm",
                "Destination": "/tmp/service_logs",
                "Mode": "",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/home/mariorodriguez/.elastic-package/tmp/service_logs/agent-sophos-utm",
                "Destination": "/run/service_logs",
                "Mode": "",
                "RW": true,
                "Propagation": "rprivate"
            }
        ],
```

Reply:
Maybe the service that was failing was because it was expecting the logs in `/run/service_logs` ? 

At the end, it looks similar the configuration shown with a `docker inspect`

Reply:
Updated settings to set explicitly read only or read write for those volumes, and keeping the same syntax in all mounts.